testdata/p4tc_samples_outputs/add_entry_1_example_control_blocks.c

```diff
@@ -113,8 +113,8 @@ if (/* hdr->ipv4.isValid() */
                                 break;
                             case MAINCONTROLIMPL_IPV4_TBL_1_ACT_MAINCONTROLIMPL_SEND_NH: 
                                 {
-                                    hdr->ethernet.srcAddr = value->u.MainControlImpl_send_nh.smac;
-                                                                        hdr->ethernet.dstAddr = value->u.MainControlImpl_send_nh.dmac;
+                                    hdr->ethernet.srcAddr = bpf_cpu_to_be64(value->u.MainControlImpl_send_nh.smac);
+                                                                        hdr->ethernet.dstAddr = bpf_cpu_to_be64(value->u.MainControlImpl_send_nh.dmac);
```

Comment:
This is intentional, we wanted to check scenario if it is not annotated, will compiler take care of version.

Reply:
I see, so this is like an error case?
I understand that it will compile find (so it wouldn't be considered an error case by the compiler), but it would not work properly in the datapath

Reply:
Yes it will be compiled properly but will be error case in datapath.